mlflow/models/evaluation/base.py

```diff
@@ -425,9 +425,50 @@ def _hash_uint64_ndarray_as_bytes(array):
     return struct.pack(f">{array.size}Q", *array)
 
 
+def _is_empty_list_or_array(data):
+    if isinstance(data, list):
+        return len(data) == 0
+    elif isinstance(data, np.ndarray):
+        return data.size == 0 or data.ndim == 0
```

Comment:
```
In [1]: import numpy as np

In [2]: np.array(1).ndim
Out[2]: 0
```

In this case, the array isn't empty, but it has ndim zero. Do we need to rename this method?

Reply:
Ahh interesting, I think we don't need the ndim check indeed.